src/main/resources/net/sourceforge/pmd/util/fxdesigner/less/tests.less

```
@@ -16,6 +16,7 @@
       .status-label {
         -fx-background-color: white;
         -fx-background-radius: 12;
+		-fx-padding: 0 1;
```

```suggestion
        -fx-padding: 0 1;
```

Checkstyle complains of the tabs